Changelog.md

```diff
@@ -34,7 +34,7 @@ __Bugfixes__
 - Fixes 301validator schematron file extension (.sch, not .xml).
 - Fixes U-factor for floors over 'other multifamily buffer space' per ENERGY STAR MFNC Rev 05.
 - Fixes modeling of 0.3 ACHnatural infiltration minimum for MF dwelling units where Aext < 0.5 and the mechanical ventilation system is solely exhaust-only.
-- Fixes ZERH Target Home and ESRD so that dual-fuel heat pumps are preserved.
+- Fixes DENH (formerly ZERH) Target Home and ESRD so that dual-fuel heat pumps are preserved.
```

Comment:
These changes don't make sense; past items in the changelog should reflect the program that was in place at the time and should never be changed. I'm going to back these out.